packages/shared/lib/services/sync/config/config.service.ts

```diff
@@ -94,7 +95,9 @@ const convertSyncConfigToStandardConfig = (syncConfigs: extendedSyncConfig[]): S
         }
     }
 
-    const { success, response: standardConfig } = convertV2ConfigObject(nangoConfig);
+    const { success, error, response: standardConfig } = convertV2ConfigObject(nangoConfig);
+
+    logger.error(`Error in converting sync config to standard config: ${error}`);
```

Comment:
there is no if to check if there is an error

Reply:
Thanks! fixed